packages/components/src/popover/README.md

```diff
@@ -143,6 +143,15 @@ If you need the `DOMRect` object i.e., the position of popover to be calculated
 - Type: `Function`
 - Required: No
 
+### toggleRef
+
+Reference to the element that's used to toggle this popover. Used to prevent `onClose` firing if focus moves to the toggle. Which usually means we clicked on the toggle. Without this, popover would reopen everytime we click on the toggle.
+
+When `toggleRef` is provided, then `onFocusOutside` prop is ignored.
```

Comment:
Why will it be ignored?

Reply:
I can't remember the reason for that to be honest. Looking at it today it totally makes sense to call this callback as well. Added it